src/styles/main.scss

```diff
@@ -1,14 +1,4 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
-
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./utils/variables";
+@import "./blocks/page";
+@import "./blocks/header";
+@import "./blocks/catalog";
```

Comment:
The import for 'catalog' should be changed to '_catalog.scss' for consistency with the naming convention used in other imported files.